RecoTracker/LSTCore/src/alpaka/PixelTriplet.h

```diff
@@ -2583,11 +2601,6 @@ namespace SDL {
                    mdsInGPU.anchorY[fourthMDIndex],
                    mdsInGPU.anchorY[fifthMDIndex]};
 
-    //get the appropriate radii and centers
-    centerX = segmentsInGPU.circleCenterX[pixelSegmentArrayIndex];
-    centerY = segmentsInGPU.circleCenterY[pixelSegmentArrayIndex];
```

Comment:
Just noticed this. Is this branch up to date with the current Tracklooper master @VourMa ? I just checked and it looks like this is removed on the current master branch.

https://github.com/SegmentLinking/TrackLooper/commit/3164f716c21d1cb80d6ebcd868eb20f625fad6a9

Reply:
I wouldn't touch those lines. This is how it should be:
![image](https://github.com/SegmentLinking/cmssw/assets/15358704/eae682d6-b070-47e3-9fff-d21bba382e72)
Check also https://github.com/SegmentLinking/cmssw/pull/40#discussion_r1636161811.

Reply:
About being up to date, please check here:
https://github.com/SegmentLinking/cmssw/issues